gsa/src/gmp/models/__tests__/settings.js

```diff
@@ -0,0 +1,60 @@
+/* Greenbone Security Assistant
+ *
+ * Authors:
+ * Steffen Waterkamp <steffen.waterkamp@greenbone.net>
+ *
+ * Copyright:
+ * Copyright (C) 2018 Greenbone Networks GmbH
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; either version 2
+ * of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
+ */
+
+import Settings from 'gmp/models/settings';
+
+describe('Settings model tests', () => {
+
+  test('should have private settings', () => {
+    const settings = new Settings({});
+
+    expect(settings._settings).toBeDefined();
```

Comment:
Please don't test internal API. Internal API is marked with the underscore. The tests should only ensure the public api is working correctly and the internal api may change every time.

Reply:
This concerns all tests in this module.

Reply:
Doing this, I could only test set() and get() within a single test, but it should work. With the last push, I also added more commits, so just that you know :)

Reply:
> I could only test set() and get() within a single test

That's completely fine.